src/info/langs/language.tera

```diff
@@ -2,142 +2,77 @@ use owo_colors::{
     AnsiColors,
     DynColors::{self, Ansi, Rgb},
 };
-use std::fmt;
-use std::fmt::Write;
-use strum::EnumIter;
 
 pub struct Colors {
     basic_colors: Vec<DynColors>,
     true_colors: Option<Vec<DynColors>>,
 }
 
-#[derive(Clone, PartialEq, Eq, Debug, clap::ValueEnum)]
-pub enum LanguageType {
-    Programming,
-    Markup,
-    Prose,
-    Data,
-}
-
-#[derive(Clone, Copy, PartialEq, Eq, Hash, EnumIter, clap::ValueEnum, Debug, Serialize)]
-#[allow(clippy::upper_case_acronyms)]
-#[clap(rename_all = "lowercase")]
-pub enum Language {
-    {% for language, attrs in languages -%}
-        {% if attrs.serialization %}#[clap(name="{{ attrs.serialization }}")]{% endif -%}
-        {{ language }},
-    {% endfor %}
-}
-
-impl fmt::Display for Language {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self {
-            {% for language, _ in languages -%}
-                Self::{{ language }} => write!(f, "{}", tokei::LanguageType::{{ language }}.name()),
-            {% endfor %}
-        }
-    }
-}
-
-impl From<tokei::LanguageType> for Language {
-    fn from(language: tokei::LanguageType) -> Self {
-        match language {
-            {% for language, _ in languages -%}
-                tokei::LanguageType::{{ language }} => Self::{{ language }},
-            {% endfor %}
-            _ => unimplemented!("Language {:?}", language),
-        }
+pub(crate) fn is_supported(language: &str) -> bool {
```

Comment:
this method could probably be extracted from the template and use make use of `get_supported_languages` internally

Reply:
>and use make use of get_supported_languages internally

I haven't benchmarked this, but its intent is to be a bit of a performance optimization, since `get_supported_langauges` returns a simple iterable, Although I haven't researched how `match` is implemented, I'm assuming it's more performant to match than iterate. Perhaps they could be combined by using a static [`IndexSet`](https://docs.rs/indexmap/latest/indexmap/set/struct.IndexSet.html) :thinking: